### ISSUE TITLE
Finetuning on custom dataset

### ISSUE THREAD

**Issue by ani0075saha** [DQiaole/ZITS_inpainting#29]
Hi,

I used your pre-trained models on my custom dataset and the inpainting results were not great.
I was thinking of finetuning the pretrained weights on my custom dataset to improve inpainting quality.

What should be the steps/commands? Which models do I need to finetune? TSR and FTR?

It would be great if you could provide some suggestions.

**Comment by DQiaole** [comment]
Hi,
I think finetune the FTR is enough, you can finetune it until the FTR converges.  The command is the same as the training command, you may need modify the config according to your datasets and mask list.
